nixos/modules/programs/mouse-actions.nix

```diff
@@ -0,0 +1,17 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
This pattern is discouraged, see #208242. Best to use `inherit` in the below `let` block.